pkg/consts/consts.go

```diff
@@ -27,7 +27,9 @@ const (
 	NetworkDataplaneCilium = "cilium"
 	NetworkDataplaneAzure  = "azure"
 
-	DefaultKubernetesMaxPods = 250
+	DefaultOverlayMaxPods    = 250
+	DefaultNodeSubnetMaxPods = 30
+	DefaultKubernetesMaxPods = 110
```

Comment:
Please add a comment on source (I presume this matches AKS defaults)

Reply:
Yes this comes directly from the AKS defaults.